### PR TITLE
Add more `@Embed` localizations to the Archetype journal and brush up Dual Weapon Reload feat

### LINKED ISSUE #17602: Clawdancer Dedication Wrong Formatting
The damage rolls on both stances seem to be incorrect on the journal entry.
image

Aditionally, also on the journal entry, the link to Claw Snag leads to the dedication instead.

### PR DESCRIPTION
**Pull request by ammalagonc**
Added missing prerequisite, also
Closes #17602

### CODE REVIEW

File: packs/feats/dual-weapon-reload.json [hunk 1]
```diff
@@ -17,7 +17,11 @@
             "value": 1
         },
         "prerequisites": {
-            "value": []
+            "value": [
```
Comment (simonward) [inline packs/feats/dual-weapon-reload.json:20]: This is a Gunslinger feat.

Comment (ammalagonc) [reply packs/feats/dual-weapon-reload.json:20]: Ah!

It got reprinted in PC2 as part of the dedication.

Fun times - do we need two versions of this feat?

Comment (simonward) [reply packs/feats/dual-weapon-reload.json:20]: I defer to @TikaelSol.

Comment (ammalagonc) [reply packs/feats/dual-weapon-reload.json:20]: Wait

We had this same issue before. It was resolved by doing something like `Gunslinger or Dual Weapon Dedication`

Check Douse Spell

Comment (TikaelSol) [reply packs/feats/dual-weapon-reload.json:20]: I remember talking about it, but don't know that Dousing Spell was the one that got the weird prereq stuff. I think technically this one may need to just get listed as an additional feat for the archetype? Assuming it is reprinted in the remaster Guns and Gears as is.

Comment (ammalagonc) [reply packs/feats/dual-weapon-reload.json:20]: I moved the dedication to the additional feats section for the moment. We can revisit this once Guns and Gears remaster hits.

**Pull request merged**